compiler/quilt/tools/util.py

```diff
@@ -350,3 +350,33 @@ def fs_link(path, linkpath, linktype='soft'):
                 os.link(str(path), str(linkpath))
         except OSError as error:
             raise QuiltException("Linking failed: " + str(error), original_error=error)
+
+def find_in_package(corenode, item):
+    """Find a (core) node in a package tree.
+    :param item: Node name or path, as in "node" or "node/subnode".
+    """
+    if not isinstance(corenode, core.Node):
+        raise QuiltException("Corenode must be a valid Quilt package node. %r" % corenode)
+
+    node = corenode
+    path = pathlib.PurePosixPath(item)
```

Comment:
I wouldn't use `pathlib` here, since this isn't an actual file path. Just split at `/`?